### ISSUE TITLE
URDF related segmentation fault

### ISSUE THREAD

**Issue by JeonghyunNam** [raisimTech/raisimLib#509]
Hi, Prof Hwangbo. Really appreciate it for sharing the masterpiece.

We are now using raisim with our customized model as below:
![image (1)](https://github.com/raisimTech/raisimLib/assets/87422054/7906eab2-14d2-4bff-bacd-94768f463517)

Training code was running well but we found some points that we mis defined for toes in our model. (We defined two geometry for toes link)
(origin)
![origin_toes_R](https://github.com/raisimTech/raisimLib/assets/87422054/063d0fa2-32f9-430b-9b3a-a94676b2e289)

and changed it to
(revised)
![toes_R](https://github.com/raisimTech/raisimLib/assets/87422054/a09b773e-8ec3-44e2-b512-8ded8399d16e)

But at this moment, we encounted the segmentation error.

We debugged it and found that it appears when vectorizedEnvironment initialized & reset the ChildrenEnvironment which is our custom env.
 
(VectorizedEnvironment.hpp)
![vecenv](https://github.com/raisimTech/raisimLib/assets/87422054/c2a08946-f9bf-4f33-9e38-fe097e9179d5)

(Result)
![checking](https://github.com/raisimTech/raisimLib/assets/87422054/d1212d18-43ac-4927-9ca6-8d24628fd63e)

We are lost since the results are different for every trials such as they sometimes reach until 100 loops.

Is there any reason about it?
Thanks for reading!

**Comment by jhwangbo** [comment]
Did you run the debug app with valgrind? It should spot the bug in most cases

**Comment by JeonghyunNam** [comment]
Thank you for advise! 
I found the error caused from the call of the getFrameOrientation method fpr links of other robots that I tested before.
So, problem solved now.

As a side note, I have a simple curiosity. the problem above also have to be caused with our original model since we are using same code that calls getFrameOrientation for wrong link.
are there any reasons for it?, or is it just a bug that I cannot found?
